wiki/Article_styling_criteria/th.md

```diff
@@ -0,0 +1,23 @@
+---
+tags:
+  - wiki standards
+---
+
+# เกณฑ์การจัดรูปแบบบทความ
+
+*สำหรับโพสต์ข่าวสาร โปรดดู: [เกณฑ์การจัดรูปแบบข่าวสาร](/wiki/News_Styling_Criteria)*
+
+เกณฑ์การจัดรูปแบบบทความ (ASC) ทำหน้าที่เป็น osu! มาตรฐานการจัดรูปแบบที่บังคับใช้ของ Wiki เพื่อรักษาความสอดคล้องในความชัดเจน การจัดรูปแบบ และเค้าโครงในบทความทั้งหมด
+
+บทความทั้งหมดควรมีจุดมุ่งหมายเพื่อให้มีไวยากรณ์ที่ถูกต้อง การสะกดคำที่ถูกต้อง และข้อมูลที่ถูกต้อง
+
+หากต้องการแนะนำการเปลี่ยนแปลงเกี่ยวกับเกณฑ์การจัดรูปแบบบทความ โปรด [เปิดปัญหาใน GitHub](https://github.com/ppy/osu-wiki/issues/new)
+
+## มาตรฐาน
+
+*หมายเหตุ: บทความด้านล่างใช้ [RFC 2119](https://tools.ietf.org/html/rfc2119) เพื่ออธิบายระดับความต้องการ*
```

Comment:
this is irrelevant: the document explains what each of the words "must", "should", etc etc. mean exactly here. I don't think that's applicable to Thai

Reply:
what you really mean here?

Reply:
https://tools.ietf.org/html/rfc2119 explains how certain key words should be interpreted. these keywords are: "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED",  "MAY", and "OPTIONAL". if your translation uses neither of these, it might be a good idea to drop this line, because it makes no sense.

Reply:
ok cool